spec/requests/requests_spec.rb

```diff
@@ -3,10 +3,16 @@
 require 'rails_helper'
 
 RSpec.describe 'Requests', type: :request do
+  # Authenticate an user
+  before do
+    user = FactoryBot.create :user
+    sign_in user
+  end
+
   describe 'GET /requests' do
     it 'works! (now write some real specs)' do
-      get requests_path
-      expect(response).to have_http_status(200)
+      # get requests_path
```

Comment:
What is the reasoning for commenting out a test? Why not remove it, if it does not make sense?

Reply:
It seems to be quite the dumb test...

> (now write some real tests)

what's that supposed to mean?

Reply:
I did not see this in the commit. The test still exists but is now uncommented.